scholarx/2022/index.html

```diff
@@ -210,12 +210,19 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentors</h1>
                                         data-target="#collapseTwo"
                                         aria-expanded="false"
                                         aria-controls="collapseTwo">
-                                    <div class="question-heading">
-                                        What are the eligibility requirements?
-                                    </div>
-                                    <div class="text-primary collapse-sign">
-                                        <i class="plusMinusSign fa fa-plus float-right"></i>
-                                    </div>
+                                        <table class="faq-table">
```

Comment:
Add `mr-4` class to the overlapping questions without this table @janithlahirukariyawasam. Try it and let me know whether it is working. I tried its worked.

Reply:
It should definitely work @janithlahirukariyawasam

Reply:
@YoshithaRathnayake please if you can show the implementation of that using screenshot?

Reply:
code?

Reply:
Yes @YoshithaRathnayake

Reply:
![image](https://user-images.githubusercontent.com/97069900/203834025-b40d9cbd-fd54-41e4-a672-d9c64c46b5ba.png)

Remove the table and add the div tags like earlier @janithlahirukariyawasam

Reply:
Thank you, I will try this and inform you.

Reply:
Sure

Reply:
@YoshithaRathnayake Yes Thank you very much!
It solves the issue.

Reply:
Your welcome @janithlahirukariyawasam

Reply:
@YoshithaRathnayake Can you check now?
Thank you very very much again.

Reply:
Sure